packages/vscode-extension/src/webview/components/shared/KyebindingInfo.tsx

```diff
@@ -0,0 +1,59 @@
+import { ReactNode, useEffect, useState } from "react";
+import "./KeybindingInfo.css";
+import { useProject } from "../../providers/ProjectProvider";
+
+interface KeybindingInfoProps {
+  commandName: string;
+}
+
+interface SymbolProps {
+  children: ReactNode;
+}
+
+const Symbol = ({ children }: SymbolProps) => {
+  return <span className="symbol">{children}</span>;
+};
+
+export const KeybindingInfo = ({ commandName }: KeybindingInfoProps) => {
+  const { project } = useProject();
+  const [keybinding, setKeybinding] = useState<string[]>([]);
+
+  function translateToUnicode(symbol: string) {
+    let icons = {
+      cmd: "\u2318", // ⌘
+      ctrl: "\u2303", // ⌃
+      alt: "\u2325", // ⌥
+      option: "\u2325", // ⌥
+      shift: "\u21E7", // ⇧
+      tab: "\u21E5", // ⇥
+      up: "\u2191", // ↑
+      down: "\u2193", // ↓
+      left: "\u2190", // ←
+      right: "\u2192", // →
```

Comment:
can't we just have the symbols in the strings here instead of unicode codes and symbols in the comments?